compiler/rustc_attr_parsing/src/parser.rs

```diff
@@ -176,7 +176,7 @@ impl<'a> ArgParser<'a> {
 pub enum MetaItemOrLitParser<'a> {
     MetaItemParser(MetaItemParser<'a>),
     Lit(MetaItemLit),
-    Err(Span, ErrorGuaranteed),
+    Err(Span),
```

Comment:
Is `MetaItemOrLitParser::Err` no longer constructed on `master`?

Reply:
I think @nnethercote 's recent work did that

Reply:
Uh, right. It was 'unintentionally' fixed in #124141 which is part of 1.88 and since we need a hotfix for 1.87 (current beta) and certainly don't want to backport #124141, we'd need a PR targeting the beta branch. Lemme think

Reply:
> we'd need a PR targeting the beta branch

Opened #140601.